docs/src/how-to/long-range.rst

```diff
@@ -0,0 +1,45 @@
+.. _userdoc-how-to-long-range-descriptor:
+
+Pure LODE long-range descriptor
```

Comment:
I'm not sure if "pure LODE" is the best way to call this descriptor. Maybe "long-range only LODE descriptor"?

Reply:
Yes, I agree.